Source Packages/java/edu/slu/tpen/servlet/UpdateAnnoListServlet.java

```diff
@@ -60,6 +62,25 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
             String content = request.getParameter("content");
             updateObject = JSONObject.fromObject(content);
             updateObject.element("TPEN_NL_TESTING", man.getProperties().getProperty("TESTING"));
+            /**
+             * This is to support reports.jsp
+             * When a user transcribes, their update line calls will contain the current folio and project.
+             * Update the folio's lastModfiedTime with Date.now (Folio.paleography)
+             * Update the project's lastModifiedFolio with the known Folio pageNumber (Project.lastModifiedFolio)
+             * Note that reports.jsp does not expose Folio.paleography.  It was already being tracked, 
+             * so I made sure to support while I was in this code.  They used it for something, not sure what. 
+             */
+            
+            if(updateObject.containsKey("currentFolio")){
```

Comment:
These are all in the referrer query string, but I do not think that is enough to stop this merge.